api/src/resources/comment.rs

```diff
@@ -237,7 +243,7 @@ const STRING_PROPERTY_PREFIX: &str = "string:";
 const NUMBER_PROPERTY_PREFIX: &str = "number:";
 
 impl Serialize for PropertyMap {
-    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+    fn serialize<S: Serializer>(&self, serializer: S) -> StdResult<S::Ok, S::Error> {
```

Comment:
What is the difference between `Result` and `StdResult`?

Reply:
Again, we're defining `Result` as our custom type alias. And we import the standard library's `Result' under a different name:
```rust
use std::result::Result as StdResult;
```